server/src/config.ts

```diff
@@ -1,6 +1,7 @@
 import boolean from "boolean";
 const devMode = boolean(get('DEV_MODE'));
-
+/* Do NOT use source-map-support in production as it uses the non-standard stack property of Errors */
```

Comment:
I was worried about this warning [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack#browser_compatibility) :

> Non-standard: This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future. 

source-map-support uses the Error.prototype.stack to make the translation between JS and TS. Although, Node.js does support Error.prototype.stack so I can enable it for production too.